doc/developer-guide.md

```diff
@@ -320,6 +320,13 @@ The builtin CMake options used when invoking CMake on the command line.
   caveats above apply, and this may result in an illegal instruction crash if
   your CPU doesn't support the generated instructions.
 
+- `CA_HOST_TARGET_<arch>_FEATURES`: This option is used by the `host`` target to
```

Comment:
LLVM's naming is a bit inconsistent but in its options this is called "target-specific attributes" and controlled by the `-mattr=` option, despite being called "features" at other levels. I expect it will be clearer to the user to use that terminology and to say that it is processed the same as that option (which I believe you are doing).